src/plugins/tokenizer.js

```diff
@@ -11,20 +11,28 @@ function defaultTokenizer(input, selection, createToken, options) {
             return token ? { id: token, text: token } : null;
         };
 
+    var duplicates = options.allowDuplicates;
```

Comment:
Please call the variable `allowDuplicates` too. It's a boolean option, but `duplicates` would indicate it's a list of duplicates.